galvan/src/main/java/org/terracotta/testing/master/AbstractHarnessEntry.java

```diff
@@ -52,8 +59,68 @@ public void runTestHarness(EnvironmentOptions environmentOptions, ITestMaster<C>
     }
   }
 
+  /**
+   * Allocate the specified number of consecutive TCP ports from a randomly chosen first port.
+   * The port range is held allocated until this {@code AbstractHarnessEntry} instance becomes
+   * weakly reachable.
+   * @deprecated Use {@code org.terracotta.utilities.test.net.PortManager.reservePorts}
+   *          in {@code org.terracotta:terracotta-utilities-port-chooser}
+   *          taking care to adopt the new usage semantics
+   */
+  @Deprecated
   public int chooseRandomPortRange(int number) {
```

Comment:
I provided an alternative implementation relying on `PortManager`, with tests, even though I deprecated the method mostly because I'm uncertain that all references to this method are identified.  If you think this is "too much", I can certainly back it out and change the method to throw instead.